backend/src/data/entities/Reminder.ts

```diff
@@ -0,0 +1,30 @@
+import { Entity, Column, ManyToOne, JoinColumn, RelationId } from 'typeorm';
+import { AbstractEntity } from '../abstract/AbstractEntity';
+import { User } from './User';
+import { Chat } from './Chat';
+
+@Entity()
+export class Reminder extends AbstractEntity {
+  @Column()
+  day: string;
```

Comment:
how will you handle two fields which are strings to show reminders?
it's hard to maintenance
you have to have only one field that is date